data/mods/RL_Classes/classes.json

```diff
@@ -249,6 +250,7 @@
         "items": {
             "both": [
                 "armor_larmor",
+                "vambraces_larmor",
```

Comment:
typo here, vambraces_larmor should be vambrace_larmor. It makes the pink text of horror at world load.